games/rogue/Portfile

```diff
@@ -21,14 +21,21 @@ homepage            http://rogue.rogueforge.net
 master_sites        ${homepage}/files/rogue5.4/
 
 checksums           md5     033288f46444b06814c81ea69d96e075 \
-                    sha1    aef9e589c4f31eb6d3eeb9d543ab8787b00fb022
+                    sha1    aef9e589c4f31eb6d3eeb9d543ab8787b00fb022 \
```

Comment:
Both `md5` and `sha1` are legacy checksum types we no longer want to continue using going forward, so we can drop those 2.

Reply:
Ok, I'll remove them